### PR TITLE
Update problem-sets-competitive-programming.md

### PR DESCRIPTION
**Pull request by Santigio**
Added a a website link to competitive programming

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Santigio** [comment]
https://exercism.org/ competitive programming

**Comment by Santigio** [comment]
new

File: more/problem-sets-competitive-programming.md [hunk 1]
```diff
@@ -32,6 +32,7 @@
 * [Dimik](https://dimikoj.com)
 * [DMOJ](https://dmoj.ca)
 * [E-olymp](https://www.e-olymp.com/en/)
+* [Exercism](https://exercism.org/)
```
Comment (metabiswadeep) [inline more/problem-sets-competitive-programming.md:35]:
```suggestion
* [Exercism](https://exercism.org)
```

**Review by metabiswadeep — CHANGES_REQUESTED** [review]

File: more/problem-sets-competitive-programming.md (hunk 1 above)
Comment (eshellman) [inline more/problem-sets-competitive-programming.md:35]:
```suggestion
* [Exercism](https://exercism.org)
```

**Pull request merged**